.github/workflows/backend.yml

```diff
@@ -37,35 +37,48 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - uses: dorny/paths-filter@v2
```

Comment:
I think I discovered in https://github.com/cfpb/consumerfinance.gov/pull/6645 that I don't think it's enough to do `@v2` and get the latest, but you'd have to do `@v2.10.2` https://github.com/dorny/paths-filter/releases